### DISCUSSION TITLE
unbuntu下无法加载插件

### DISCUSSION THREAD

**Discussion by xh22805** [nonebot/discussions#211] [问答]
![图片](https://user-images.githubusercontent.com/104178374/191533161-eaa358f7-9eef-437e-91cc-2a79b1efd8cc.png)
![图片](https://user-images.githubusercontent.com/104178374/191534319-8bf9f2f9-76ac-412e-9e87-6afdcf9cd265.png)
![图片](https://user-images.githubusercontent.com/104178374/191534639-f580015f-d28f-489c-9394-499810a12e79.png)
文件是从windows上搬过去的，在windows上正常，unbuntu上能连接go-cqhttp，但是无法加载插件，echo命令也无反应。
系统unbuntu server 22.0.4
python Version: 3.1.0
Nonebot Version: 2.0.0-beta4

**Comment by yanyongyu** [comment]
你的运行路径不对吧
![image](https://user-images.githubusercontent.com/42488585/191536540-72ab21c8-57a5-4e5b-9e16-784f6f03c6f1.png)

**Comment by xh22805** [comment]
![图片](https://user-images.githubusercontent.com/104178374/191537884-d1a4c36b-8c98-4436-8f41-ea411dda1d72.png)
不能这样写吗？我去文件目录下试试...

**Comment by xh22805** [comment]
可以了，谢谢大佬
